rules/LeagueEvent/Rector/MethodCall/DispatchStringToObjectRector.php

```diff
@@ -94,12 +97,7 @@ private function shouldSkip(MethodCall $methodCall): bool
         ])) {
             return false;
         }
-
-        if (! $this->getStaticType($methodCall->args[0]->value) instanceof StringType) {
-            return true;
-        }
-
-        return true;
+        return $this->getStaticType($methodCall->args[0]->value) instanceof StringType;
```

Comment:
@lulco this seems always return true, should just return `! $this->nodeTypeResolver->isObjectTypes()` above cond.

Reply:
ref the original code that always return true: https://github.com/rectorphp/rector/blob/4b71eaa53d16c9c8cf4a6a2eb1fe4bf6d8006c58/rules/LeagueEvent/Rector/MethodCall/DispatchStringToObjectRector.php#L96-L100